docs/articles/dockerfile_best-practices.md

```diff
@@ -146,16 +146,33 @@ with backslashes.
 Probably the most common use-case for `RUN` is an application of `apt-get`.
 When using `apt-get`, here are a few things to keep in mind:
 
-* Don’t do `RUN apt-get update` on a single line. This will cause
-caching issues if the referenced archive gets updated, which will make your
-subsequent `apt-get install` fail without comment.
-
 * Avoid `RUN apt-get upgrade` or `dist-upgrade`, since many of the “essential”
 packages from the base images will fail to upgrade inside an unprivileged
 container. If a base package is out of date, you should contact its
 maintainers. If you know there’s a particular package, `foo`, that needs to be
 updated, use `apt-get install -y foo` and it will update automatically.
 
+* Don’t use `RUN apt-get update` on a single line. This will cause
```

Comment:
I see this was already like this, but I think we should avoid "on a single line", because readers might confuse that with "you should wrap this line".

Thinking what would be a better way to describe this, perhaps something with "Don't put `apt-get update` in a separate `RUN` statement".

I'd like to defer this to @moxiegirl though. :relaxed: